app/src/main/java/com/mmm/parq/fragments/DriverSettingsFragment.java

```diff
@@ -19,7 +23,25 @@
     private RecyclerView mRecyclerView;
     private ItemAdapter mItemAdapter;
 
-    public DriverSettingsFragment() {}
+    private View.OnClickListener mOnClickListener;
+
+    public DriverSettingsFragment() {
+        mOnClickListener = new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                int itemPosition = mRecyclerView.getChildAdapterPosition(v);
+
+                // TODO(matt): figure out a better way to differentiate clicks; doesn't scale
+                switch (itemPosition) {
```

Comment:
So an alternative is just to cast v to a TextView, and then switch based upon the text it contains. 

So something like:
```
String itemText = ((TextView) v).getText()
switch(itemText) {
  case "Logout":
    blah
}
```

Not sure if there's really that much benefit to that though, all it really does is make it a little easier to know what action each case corresponds to